app/src/main/java/org/liberty/android/fantastischmemo/utils/NotificationUtil.java

```diff
@@ -37,6 +42,18 @@ public NotificationUtil(@ForApplication Context appContext) {
         this.appContext = appContext;
     }
 
+    public void createNotificationChannel() {
+        // Create the Notification Channel, but only on API 26+
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+            CharSequence name = appContext.getString(R.string.notification_text);
```

Comment:
I would suggest putting a more sensible name for the channel name like "Review reminder", the name will show up in settings for people to toggle.

Reply:
Good point. I had only checked the German string for R.string.notification_text ("Erinnerungen"), which can be translated to "Reminders". I didn't notice that the English string is actually "Notification".

I've added better names as you suggested.